src/axom/primal/tests/primal_clip.cpp

```diff
@@ -729,34 +729,56 @@ TEST(primal_clip, tet_tet_clip_nonintersect)
 {
   using namespace Primal3D;
 
-  TetrahedronType tet1(PointType({-1, -1, -1}),
-                       PointType({-1, 0, 0}),
-                       PointType({-1, -1, 0}),
-                       PointType({0, 0, 0}));
-
-  TetrahedronType tet2(PointType({1, 0, 0}),
-                       PointType({1, 1, 0}),
-                       PointType({0, 1, 0}),
-                       PointType({1, 0, 1}));
+  {
+    TetrahedronType tet1(PointType {-1, -1, -1},
+                         PointType {-1, 0, 0},
+                         PointType {-1, -1, 0},
+                         PointType {0, 0, 0});
+    EXPECT_TRUE(tet1.signedVolume() > 0.);
+
+    TetrahedronType tet2(PointType {1, 0, 0},
+                         PointType {1, 1, 0},
+                         PointType {0, 1, 0},
+                         PointType {1, 0, 1});
+    EXPECT_TRUE(tet2.signedVolume() > 0.);
+
+    PolyhedronType poly = axom::primal::clip(tet1, tet2);
+    EXPECT_EQ(0.0, poly.volume());
+  }
 
-  PolyhedronType poly = axom::primal::clip(tet1, tet2);
-  EXPECT_EQ(0.0, poly.volume());
+  // User-provided test case; tetrahedra do not overlap
+  {
+    TetrahedronType tet1(PointType {0, 1, 0},
+                         PointType {-1, 0, 0},
+                         PointType {0, -1, 0},
+                         PointType {-0.25, 0, 0.25});
+    EXPECT_TRUE(tet1.signedVolume() > 0.);
+
+    TetrahedronType tet2(PointType {0.74899999999999922, 0, 1},
+                         PointType {1.7489999999999992, 0, 0},
+                         PointType {0.74899999999999922, -1, 0},
+                         PointType {0.99899999999999922, 0, 0.25});
+    EXPECT_TRUE(tet2.signedVolume() > 0.);
+
+    PolyhedronType poly = axom::primal::clip(tet1, tet2);
+    EXPECT_EQ(0.0, poly.volume());
+  }
 }
```

Comment:
This uses the test data provided by @liyangrock in https://github.com/LLNL/axom/pull/1005 for non-intersecting tetrahedra using `primal::clip(tet, tet)`.